frontend/src/config/index.ts

```diff
@@ -12,10 +16,3 @@ axios.interceptors.response.use(
     return Promise.reject(error);
   },
 );
-if (!import.meta.env.hasOwnProperty('VITE_BASE_API_URL')) {
```

Comment:
Is this on purpose?

Reply:
Yes, I moved the environment variables into their own file `env.ts` because I need the environment variables already loaded for `i18n` initialization

Reply:
Please document somewhere how the bootstrapping of the application works (which is done in which order).